source/help/getting-started/searching.rst

```diff
@@ -1,24 +1,17 @@
 Searching
 =========
 
---------------
+Use the search box to find messages and replies that match the search terms that you enter. All channels that you are a member of are searched.
```

Comment:
We might want to include a note that only the team that you are currently viewing is  searched

e.g. if you're viewing team A, only channels (that you're a member of) are searched within team A, but not channels in other teams.